docs/source/message_matcher.rst

```diff
@@ -67,6 +67,9 @@ Message Variables
     - **Timestamp**
     - **Severity**
     - **Pid**
+- DateTime
```

Comment:
We don't actually have a DateTime data type.  I would just update the existing entry to:

**Timestamp** (Nanoseconds since the Unix epoch or a quoted RFC3339 string that will be converted to nanoseconds e.g., '2014-02-03T14:02:03Z')